### PR TITLE
Updates as per discussion in the website call

### PR DESCRIPTION
**Pull request by UtkarshMishra12**
Signed-off-by: Utkarsh Mishra <nasautkarsh@gmail.com>

**Description**
Updates as per discussion in the website call 

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: site/src/components/Navbar/index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const Navbar = () => {
       </Toggle>
       <Navbox open={navbarOpen}>
         <NavItem to="/chapters">Book Chapters</NavItem>
-        <NavItem to="/authors">Meet the Authors</NavItem>
+        <NavItem to="/authors">Meet our Authors</NavItem>
```
Comment (leecalcote) [inline site/src/components/Navbar/index.js:23]: "the" is fine and is most appropriate.

**Review by leecalcote — COMMENTED** [review]
Other than reverting "the", the rest looks good.

File: site/src/components/Navbar/index.js (hunk 1 above)
Comment (UtkarshMishra12) [reply site/src/components/Navbar/index.js:23]: On it

**Comment by l5io** [comment]
🚀 Preview for commit 78ca873593cec8cd0dcb6d451110b168aa00d694 at: https://6361234a163a55371954355a--service-mesh-patterns.netlify.app

**Comment by UtkarshMishra12** [comment]
@leecalcote Changed 🙌

**Comment by l5io** [comment]
🚀 Preview for commit 0983fd65382fac8ed6b554b32bb393ba9ab12653 at: https://6361248406aa033c0ad39c25--service-mesh-patterns.netlify.app

**Review by leecalcote — APPROVED** [review]
LGTM

**Review by Chadha93 — APPROVED** [review]
LGTM

**Comment by l5io** [comment]
🚀 Preview for commit d492b413f209074d75a9ad38b64dc68e94df00c8 at: https://63612564d17e5d3d3af582a7--service-mesh-patterns.netlify.app

**Pull request merged**